CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 To publish a new release run `scripts/release` from the project directory.
 
 ## [Unreleased]
-
+- Support for [foot](https://codeberg.org/dnkl/foot/) (see [GH-190]) 
```

Comment:
Please follow the changelog format here, by adding a corresponding section header.